.github/CODEOWNERS

```diff
@@ -1462,6 +1473,7 @@ x-pack/test/api_integration/apis/management/index_management/inference_endpoints
 /x-pack/test/security_solution_api_integration @elastic/security-solution
 /x-pack/test/api_integration/apis/security_solution @elastic/security-solution
 #CC# /x-pack/plugins/security_solution/ @elastic/security-solution
+/x-pack/test/functional/es_archives/cases/signals/duplicate_ids @elastic/security-solution @elastic/security-threat-hunting
```

Comment:
This is consumed by `cases_api_integration/` , I think it makes sense to be owned by @elastic/response-ops

Reply:
https://github.com/elastic/kibana/pull/194411/commits/748578a7931c840bce29bf63e5e85ae93c5749c6